lib/private/validate-configured-input-values.js

```diff
@@ -45,6 +45,35 @@ module.exports = function validateConfiguredInputValues(machine){
         );
         return;
       }
+      // For inputs with string-type examples (excluding * and ===, which have special meaning), don't allow empty string as a value.
+      if (_.isString(inputDef.example) && inputDef.example !== '*' && inputDef.example !== '===' && machine._configuredInputs[inputName] === '') {
```

Comment:
@sgress454 Whoops nevermind-- i misread this.  What you have is correct.  (The lamda type doesn't allow null)